test/csuite/timestamp_abort/main.c

```diff
@@ -105,6 +105,13 @@ static int recover_and_verify(uint32_t backup_index, uint32_t workload_iteration
 extern int __wt_optind;
 extern char *__wt_optarg;
 
+static struct {
+    bool tmr_ready_to_go;
+    WT_CONDVAR *cond_tmr_thrd;
+    bool ckpt_ready_to_go;
+    WT_CONDVAR *cond_ckpt_thrd;
+} thrd_sync_conds = {false, NULL, false, NULL};
```

Comment:
Let's keep `checkpoint` as `ckpt`, but we can keep the full words for `timer` and `thread.
```suggestion
static struct {
    bool timer_ready_to_go;
    WT_CONDVAR *timer_cond;
    bool ckpt_ready_to_go;
    WT_CONDVAR *ckpt_cond;
} thread_sync_conds = {false, NULL, false, NULL};
```